### PR TITLE
feat(code): add vscode-arm.deb

### PR DESCRIPTION
**Pull request by fsx950223**
Fixes #

(let GitHub automatically close an issue when this pull request gets merged)
add vscode-arm binary install package
## Before you submit a pull request
This template is not neccessary when you do simple things like updating packages to the latest version. But in doubt, it's always better so provide some information.

## Description
Provide a description, what your changes do and why they are important

Please link issues and other pull requests connected to this one.

## Addtional information
Mention things we might need to know. Like:

Works properly:
- [x] x86_64
- [ ] aarch64 (reasons why it doesn't)

---

## That's it
Thank you for submitting your pull request.
When done, please delete the parts of this template which you don't need or these, which are only for guidance.

### CODE REVIEW

File: packages/code.rb [hunk 1]
```diff
@@ -25,7 +26,12 @@ class Code < Package
   depends_on 'libgconf'
```
Comment (uberhacker) [inline packages/code.rb]: You will need `depends_on 'dpkg'` since this package uses that executable.

File: packages/code.rb [hunk 2]
```diff
@@ -17,6 +17,7 @@ class Code < Package
   end
 
   binary_url ({
+    armv7l:'https://github.com/stevedesmond-ca/vscode-arm/releases/download/1.28.0/vscode-1.28.0.deb'
```
Comment (uberhacker) [inline packages/code.rb]: Are you sure this works?  This section expects a Chromebrew packaged archive.

File: packages/code.rb [hunk 3]
```diff
@@ -25,7 +26,12 @@ class Code < Package
   depends_on 'libgconf'
   depends_on 'xdg_base'
   depends_on 'sommelier'
-
+  def self.preinstall
+    case ARCH
+    when 'armv7l'
+      system "dpkg", "-i", "vscode-1.28.0.deb"
```
Comment (uberhacker) [inline packages/code.rb]: This technically should go in the `def self.install` section.

**Comment by fsx950223** [comment]
There is too much problems

**Pull request closed without merging**